src/foam/nanos/notification/email/services.jrl

```diff
@@ -4,7 +4,7 @@ p({"class":"foam.nanos.boot.NSpec", "name":"emailTemplateDAO",                 "
 
 p({"class":"foam.nanos.boot.NSpec","name":"email","serviceScript":"import foam.nanos.notification.email.NullEmailService;\nimport foam.nanos.notification.email.ProxyEmailService;\nreturn new ProxyEmailService.Builder(x)\n    .setDelegate(new NullEmailService.Builder(x).build())\n    .build();"})
 
-p({"class":"foam.nanos.boot.NSpec", "name":"localEmailMessageDAO",                  "lazy":true,   "serve":false, "serviceScript":"return new foam.dao.EasyDAO.Builder(x).setOf(foam.nanos.notification.email.EmailMessage.getOwnClassInfo()).setAuthenticate(false).setCreatedByAware(true).setCreatedAware(true).setSeqNo(true).setDecorator(new foam.dao.FixedSizeDAO.Builder(x).build()).setInnerDAO(new foam.dao.WriteOnlyJDAO(x, foam.nanos.notification.email.EmailMessage.getOwnClassInfo(), \"emails\")).build()"})
+p({"class":"foam.nanos.boot.NSpec", "name":"localEmailMessageDAO",                  "lazy":true,   "serve":false, "serviceScript":"return new foam.dao.EasyDAO.Builder(x).setOf(foam.nanos.notification.email.EmailMessage.getOwnClassInfo()).setAuthenticate(false).setCreatedByAware(true).setCreatedAware(true).setSeqNo(true).setJournalType(foam.dao.JournalType.SINGLE_JOURNAL).setJournalName(\"emails\").build()"})
```

Comment:
Should be emailMessages to fit the naming convention since is an emailMessageDAO not an emailDAO